lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -795,6 +796,65 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
     """
   end
 
+  @doc """
+  Chart component
+
+  You can see it in use in Metrics page.
+  """
```

Comment:
Before merging, you probably want to look at the documentation, including attrs.